extmod/modussl_mbedtls.c

```diff
@@ -173,7 +175,7 @@ STATIC mp_obj_ssl_socket_t *socket_new(mp_obj_t sock, struct ssl_args *args) {
     mbedtls_ctr_drbg_init(&o->ctr_drbg);
     #ifdef MBEDTLS_DEBUG_C
     // Debug level (0-4) 1=warning, 2=info, 3=debug, 4=verbose
-    mbedtls_debug_set_threshold(0);
+    mbedtls_debug_set_threshold(3);
```

Comment:
why this change?

Reply:
Debugging is already normally disabled by the use of `MBEDTLS_DEBUG_C`, so there's no need to also set the default level to `0`. If someone is enabling debugging, they probably want _some_ amount of useful output by default. This just seemed like a sensible default level.

Reply:
OK